apps/src/sites/studio/pages/admin_standards/index.js

```diff
@@ -2,7 +2,7 @@ $(document).ready(function() {
   $('#import-standards').click(function(e) {
     var script = $('#select option:selected').val();
     var url =
-      'http://www.codecurricula.com/metadata/' + script + '/standards.json';
```

Comment:
@bethanyaconnor codecurricula isn't really an http it's https, but I was asking for http and it still worked, so I don't actually think there's a larger issue there with curriculum builder. Sorry for the confusion!